README.md

```diff
@@ -5,10 +5,9 @@
 
 ## URLs
 
-- :tada: https://api.mynotif.fr/
-- :memo: https://api.mynotif.fr/swagger/
-- :memo: https://api.mynotif.fr/redoc/
-- :goal_net: https://sentry.io/organizations/andre-5t/issues/3096848907/?project=6257099
```

Comment:
Sentry is actually still valid, but it's no a team account yet.
As for the other links I still have some hopes we can move them over at some point.
We need to see if route53 support .fr domains unlike Cloudflare

Reply:
oki